### ISSUE TITLE
Masked MI loss

### ISSUE THREAD

**Issue by LYL1946734397** [Guanys-dar/MGDN#1]
Thank you for sharing the code, but I couldn't find the code for the Masked MI loss mentioned in the paper. So, I wanted to ask if it's my oversight or if this part of the code hasn't been made public yet.

**Comment by Guanys-dar** [comment]
Hi, the training code (including the proposed masked MI loss) is still reorganized.
Our implementation of the masked MI loss heavily depends on [Man Zhou`s implementation.](https://github.com/manman1995/Mutual-Information-driven-Pan-sharpening/blob/5915c65f3f24ddfcf5d528d80d7b35b7f2ae9de3/models/GPPNN.py#L334)
